wire-ios-sync-engine/Source/Synchronization/ZMOperationLoop+PushChannel.swift

```diff
@@ -21,12 +21,23 @@ import Foundation
 extension ZMOperationLoop: ZMPushChannelConsumer {
 
     public func pushChannelDidReceive(_ data: ZMTransportData) {
-        Logging.network.info("Push Channel:\n\(data)")
-
         if let events = ZMUpdateEvent.eventsArray(fromPushChannelData: data), !events.isEmpty {
             Logging.eventProcessing.info("Received \(events.count) events from push channel")
             events.forEach({ $0.appendDebugInformation("from push channel (web socket)")})
-            self.updateEventProcessor.storeAndProcessUpdateEvents(events, ignoreBuffer: false)
+
+            if syncStatus.isSyncing {
+                let groups = syncMOC.enterAllGroupsExceptSecondary()
```

Comment:
What does this mean?

Reply:
so we use dispatchGroup to wait for integration tests. we exclude the second one because it is used for enqueueDelayedSaving and it will mess up the tests if you enter in this one